modules/installation-configuration-parameters.adoc

```diff
@@ -320,6 +324,23 @@ control plane MachinePool.
 |The AWS region that the installation program creates control plane resources in.
 |Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
 
+|`platform.aws.amiID`
```

Comment:
amiID is required parameters, choose one of following options as required:
option 1: platform.aws.amiID 
option 2: compute.platform.aws.amiID and controlPlane.platform.aws.amiID